static/js/client.js

```diff
@@ -124,6 +149,10 @@ var app = {}, browserSettings = {}, browserStorage = $.localStorage;
         bg_title = s(scaleBg(currentMgdl)) + s(deltaDisplay) + s(direction.info(latestSGV).label) + bg_title;
       }
     }
+    } else {
+    	bg_title = message.title + ' ' + message.message;
+    	$('h1.customTitle').text(bg_title);
+    }
 
     $(document).attr('title', bg_title);
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['bg_title' used out of scope.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/768506908/issues/source?bid=2452379&fileBranchId=2457543#l157)